lib/python/picongpu/picmi/diagnostics/openpmd.py

```diff
@@ -36,31 +36,32 @@ class OpenPMD:
     range: str, optional
         Contiguous range of cells per dimension to dump (e.g., ":,:,:").
         Format: comma-separated ranges like "begin:end,begin:end,begin:end".
-        Default: None (all cells).
+        Default: ":,::,:" (all cells).
```

Comment:
I think this is a typo, right? 
```suggestion
        Default: ":,:,:" (all cells).
```